client/src/components/screens/PatientsPrescriptionsSearch/SidebarFilters.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import intl from 'react-intl-universal';
+import { useHistory } from "react-router-dom";
+import { InfoCircleOutlined, ReadOutlined } from '@ant-design/icons';
+import { ISqonGroupFilter } from '@ferlab/ui/core/data/sqon/types';
+import { Col, Row, Tooltip } from 'antd';
+
+import { ExtendedMappingResults, usePrescriptionSearch } from 'store/graphql/prescriptions/actions';
+import { Results } from 'store/graphql/prescriptions/models';
+import { generateFilters } from 'store/graphql/utils/Filters';
+
+import SearchBar from './SearchBar';
+import { MAX_NUMBER_RESULTS } from './SearchPrescription';
+
+import style from './SidebarFilter.module.scss';
+
+export type SidebarFilterProps = {
+  filters: ISqonGroupFilter;
+  results: Results;
+  extendedMapping: ExtendedMappingResults;
+};
+
+export interface ItemProps {
+  label: React.ReactElement;
+  value: string;
+}
+
+const sqon = {
+  content: [],
+  op: 'and',
+};
+
+const SidebarFilters = ({
+  extendedMapping,
+  filters,
+  results
+}: SidebarFilterProps): React.ReactElement => {
+  const data = results;
+  const options: ItemProps[] = [];
+  const history = useHistory();
+
+  const prescriptions = usePrescriptionSearch({
+    first: MAX_NUMBER_RESULTS,
+    offset: 0,
+    sqon: sqon,
+  });
+
+  if (prescriptions && prescriptions.data) {
```

Comment:
I think it could be placed in a component of its own where you passed prescriptions.